app-office/fcrm/additional-files/fcrm.rdef.in

```diff
@@ -0,0 +1,40 @@
+resource app_signature "application/x-vnd.F-crm";
+
+resource app_flags B_SINGLE_LAUNCH;
+
+resource app_version {
+	major  = 0,
+	middle = 2,
+	minor  = 0,

```

Comment:
As you use sed to set the versions here maybe:
```
	major  = @MAJOR@,
	middle = @MIDDLE@,
	minor  = @MINOR@,
```
Check other recipes how they do it

Reply:
ok  done , thanks